src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,22 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
```

Comment:
Consider reviewing the cascade type used here. While `CascadeType.PERSIST` is suitable for persisting new entities, you might want to consider other cascade types depending on your application's requirements, such as `MERGE` or `REMOVE`, if you need those operations to cascade as well.